.buildkite/pipeline.yml

```diff
@@ -1,6 +1,7 @@
 steps:
-  - label: "Tests"
+  - name: ":mocha: Test"
+    command: npm test
     plugins:
-      - docker-compose#v2.0.0:
+      - docker-compose#v3.9.0:
```

Comment:
May as well go all the way to https://github.com/buildkite-plugins/docker-compose-buildkite-plugin/releases/tag/v4.16.0?

Reply:
Oh that's what I meant to do! I think I looked at the node docker example and got mixed up.